schema.sql

```diff
@@ -0,0 +1,11 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals
+(
+    id INTEGER,
```

Comment:
- [ ] [OPTIONAL] When creating a table, I think it is a better idea to make the `id` a primary key and auto increment. That way you don't have to input the ID value when creating new entries in the database. You can do the following to achieve this. 😎️

```
id SERIAL PRIMARY KEY,
```